catalog/spatial/geocoding/spatial-geocoding-offline-catalog/src/main/java/org/codice/ddf/spatial/geocoding/index/GeoNamesCatalogIndexer.java

```diff
@@ -360,4 +363,9 @@ private void fileProcessingComplete(String resource) {
       LOGGER.debug("Unable to create {} to indicate {} processed", processedIndicator, resource, e);
     }
   }
+
+  private long getSortPopVal(long popSize) {
+    // gives a sorting score from 2-10 based on population
+    return Math.min(10, (popSize - 1) / 50_000 + 2);
```

Comment:
✏️ I still don't quite know what these numbers do, but I will approve.

Reply:
does this mean everything above 400k is a tie?  So depending on if the data uses city limits or metro population, London Ontario and London UK may be a tie?